test/fixes/issue-09.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var test = require('tape');
+var parse = require('../..');
+
+test('Issue #9 fixed', function (t) {
```

Comment:
the test name should describe what it's testing; "which issue it is" can be found via the git log :-)

Reply:
its under the **string and alias** group now